conf/server.yaml.template

```diff
@@ -4,15 +4,15 @@ SERVER:
   #  - replace.with.satellite.hostname
   #  - replace.with.satellite.hostname
   VERSION:
-    # The full release version (6.9.2)
-    RELEASE: 6.9.2
+    # The full release version (6.15.0)
+    RELEASE: 6.15.0
     # The snap version currently testing (if applicable)
     SNAP: 1.0
     # The source of Satellite packages. Can be one of:
-    # internal, ga, beta
+    # internal, ga
     SOURCE: "internal"
     # The RHEL Base OS Version(x.y) where the Satellite is installed
-    RHEL_VERSION: '7'
+    RHEL_VERSION: '8'
```

Comment:
Why not RHEL9 at least for master/6/16/0?

Reply:
it's just a template ;-) but I bumped the version :-)